src/components/chat/ask/chatInput.tsx

```diff
@@ -50,11 +52,11 @@ const ChatInput = () => {
     ]);
 
     // TODO: handle errors
-    const responseMessage = await submitUserMessage(question, focus);
+    const responseMessage = await submitUserMessage(question, aiState.focus);
```

Comment:
not sure i completely trust it, but from testing it seems like in the action when it does `getMutableAIState` the changes to focus are reflected. 
```js
export const submitUserMessage = async (userInput: string, focus: Focus) => {
  // provided by <AI> in the page.tsx
  const aiState = getMutableAIState<typeof AI>();
  console.log('aiState.focus: ', aiState.get().focus, ' focus: ', focus);
  ```
  if we still wanna pass it directly, i think right at the top we should update the AI state. currently we do it on line 92 when we do the messages, but i'd rather be on the safe side and not do any work where they could be out of alignment 
  ```js
    aiState.update({
      ...aiState.get(), // blank state
      // id is set on save to db
      focus, // focus from the user
    });
```